rs_scraper.py

```diff
@@ -27,71 +26,84 @@ def __str__(self):
             text += "---\n"
         return text
 
-    def create_tree(self, stream, format='url'):
-        #creates a tree object for XPath queries. can be created from an HTML file or a webpage url
-        #
-        #     inputs  :  type  : required  :  default  :  possible values
-        #     stream  :  str   :    yes    :           :
-        #     format  :  str   :           :  'url'    : 'url', 'file'
-        #
-        if (format == 'url'):
-            page = requests.get(stream, headers={'content-type' : 'application/json'})
-            tree = html.fromstring(page.content)
-        elif (format == 'file'):
-            #TODO: lxml library probably has a way to this
-            with open(stream, 'r') as f:
-                page = f.read()
-            tree = html.fromstring(page)
-        return tree
-
-    def scrape(self, tree):
-        #scrape content off of html tree
-        #
-        #     inputs  :     type     : required  :  default  :  possible values
-        #     tree    :  lxml.tree   :    yes    :           :
-        #
-
-        self.scrape_usernames(tree)
-        num_posts = self.get_num_posts(tree)
-
+    def __create_tree__(self, page_num):
+        """
+        creates a tree object for XPath queries and saves it to self.current_tree
+            inputs   :  type  : required  :  default  :  possible values
+            page_num : int    : yes       :           :
+        """
+        page = requests.get(self.thread.format(page_num), headers={'content-type' : 'application/json'})
+        tree = html.fromstring(page.content)
+        self.current_tree = tree
+
+    def scrape_page(self, page_index, post_index=1):
+        """
+        !!!NOTE: by deaful this method will return the entire page. By passing a page_index, it will return less!!!!
```

Comment:
spelling. also did you mean `post_index` 
```suggestion
        !!!NOTE: by default this method will return the entire page. By passing a page_index, it will return less!!!!
```